04-files-from-grid.md

```diff
@@ -34,8 +34,8 @@ quick look at the dataset, it is sufficient to just obtain one of those files.
 
 Before we can download the file, we need to set up our connection with the grid and load the Dirac software:
 ```bash
-lhcb-proxy-init
 lb-run LHCbDirac $SHELL
+lhcb-proxy-init
```

Comment:
Why not, but then bellow the lb-run should be added again for the dirac-dms-get-file. 
Do you prefer it this way ?